File: app/src/main/java/org/tasks/ui/EmptyTaskEditFragment.java
package org.tasks.ui;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import org.tasks.R;

public class EmptyTaskEditFragment extends Fragment {

  @Nullable
  @Override
  public View onCreateView(
      LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
    return inflater.inflate(R.layout.fragment_task_edit_empty, container, false);
  }
}

File: app/src/main/java/org/tasks/ui/EmptyTaskEditFragment.kt
package org.tasks.ui

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import org.tasks.databinding.FragmentTaskEditEmptyBinding

class EmptyTaskEditFragment : Fragment() {

    companion object {
        fun newEmptyTaskEditFragment(): EmptyTaskEditFragment {
            return EmptyTaskEditFragment()
        }
    }

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?
    ): View {
        return FragmentTaskEditEmptyBinding.inflate(inflater).root
    }
}